deploy/kubernetes/docker/build.sh

```diff
@@ -138,7 +138,6 @@ cp "$RSS_DIR/$RSS_FILE" .
 GIT_BRANCH=$(git rev-parse --abbrev-ref HEAD)
 GIT_COMMIT=$(git describe --dirty --always --tags | sed 's/-/./g')
 IMAGE_VERSION=$(cat $RSS_DIR/version.txt | awk '{$1=$1; print}' | sed '/^$/d')
-IMAGE=$REGISTRY/rss-server:$IMAGE_VERSION
```

Comment:
why remove it?

Reply:
It is redudant. Being redeclared a line after